src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClientX/Handlers/DataSourceParsingHandler.cs

```diff
@@ -12,13 +12,10 @@ namespace Microsoft.Data.SqlClientX.Handlers
     /// <summary>
     /// Handler to populate data source information required for transport handler
     /// </summary>
-    internal class DataSourceParsingHandler : IHandler<ConnectionHandlerContext>
```

Comment:
This should be in Microsoft.Data.SqlClientX.Handlers.Connection namespace, considering data source parsing is only applicable to Connection flow.